torch/quantization/fx/quantization_patterns.py

```diff
@@ -290,10 +290,10 @@ def convert(self,
         if is_reference and self.binary_op in binary_reference_op_supported_dtypes and \
                 dtypes in binary_reference_op_supported_dtypes[self.binary_op]:
             if dtypes in binary_op_int8_dtypes:
-                args = load_arg(quantized=[0, 1])(node.args)
-                args = load_arg(quantized=False)(node.args)
-                kwargs = load_arg(quantized=False)(node.kwargs)
-                op_out = quantized_graph.node_copy(node, load_arg(quantized=False))
+                args = load_arg(quantized=[torch.quint8, torch.qint8])(node.args)
```

Comment:
would it make sense to rename the argument to something like `dtype=[torch.quint8, torch.qint8]` instead of keeping it named `quantized`?

Reply:
yes, agree, we can do this in a separate PR I think

Reply:
sure, sg